javascriptv3/example_code/lambda/README.md

```diff
@@ -1,30 +1,63 @@
-# AWS Lambda JavaScript SDK v3 code examples
-AWS Lambda lets you run code without provisioning or managing servers. You pay only for the compute time you consume.
+# AWS Lambda code examples for the AWS SDK for JavaScript in Node.js
```

Comment:
I thought you/we decided to drop the "in Node.js" part of the SDK name? I could be wrong... 🤷‍♀️

Reply:
We left it undecided: https://github.com/awsdocs/aws-doc-sdk-examples/pull/3506#discussion_r954933005